### PR TITLE
Refactor improve structure

### PR DESCRIPTION
**Pull request by anhfactor**
move i18n to share "@libs/i18n"
add Makefile & update Readme

### CODE REVIEW

**Comment by DistractionBoy** [comment]
could you make the `translate()` function be what's returned from a hook called "useTranslate" instead of passing it down in the component tree? we do have some time right now

**Comment by anhfactor** [comment]
> could you make the `translate()` function be what's returned from a hook called "useTranslate" instead of passing it down in the component tree? we do have some time right now

okay, let me make it.

**Comment by anhfactor** [comment]
> could you make the `translate()` function be what's returned from a hook called "useTranslate" instead of passing it down in the component tree? we do have some time right now

done, you can review it.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,35 @@ If your toolchain supports incremental builds, you can also use `rush build` to
...
+├── projects (nextjs) 
+│   ├── aggregate       
+│   ├── creators        
+│   └── marketing-site  
+├── libs
+│   ├── i18n          (locales for multi-language support)
+│   └── ui            (webpack, share stub components)
+└── common (rush config)
+```
+
```
Comment (DistractionBoy) [inline README.md:82]: thanks for this

**Review by DistractionBoy — APPROVED** [review]
LGTM

**Pull request merged**